src/rdkafka.h

```diff
@@ -56,45 +56,48 @@ extern "C" {
 #endif
 
 #ifdef _WIN32
-#include <basetsd.h>
-#ifndef WIN32_MEAN_AND_LEAN
-#define WIN32_MEAN_AND_LEAN
-#endif
-#include <winsock2.h>  /* for sockaddr, .. */
-#ifndef _SSIZE_T_DEFINED
-#define _SSIZE_T_DEFINED
+#  include <basetsd.h>
```

Comment:
Please avoid whitespace/reformat changes (here and in rdkafkacpp.h)

Reply:
Ah. sorry. I had a hard time matching the blocks, so added indentation to help readability. Is it OK to leave it in?

Reply:
Please restore the original indent so this PR only changes what it actually aims to change.

We'll address style when we have a proper C style checker.

Reply:
done